src/patternfly/components/DataList/examples/index.js

```diff
@@ -33,6 +37,13 @@ export default (props) => {
       <Example heading="Data list simple" handlebars={DataListSimpleExampleRaw} docs={DataListSimpleExampleDoc}>
         {dataListSimpleExample}
       </Example>
+      <Example
+        heading="Data list simple h2: screen reader accessible"
```

Comment:
I suggest naming this example something like "Data list simple with headings", and then leave it to the documentation to explain why headings are beneficial.

Reply:
sure thing!